src/api/faucet.ts

```diff
@@ -24,7 +24,7 @@ export class Faucet {
    * @param timeoutSecs Timeout in seconds. Defaults to 20 seconds.
    * @returns Hashes of submitted transactions
    */
-  async fundAccount(args: { accountAddress: HexInput; amount: number; timeoutSecs?: number }): Promise<Array<string>> {
+  async fundAccount(args: { accountAddress: AccountAddress; amount: number; timeoutSecs?: number }): Promise<Array<string>> {
```

Comment:
This should be consistent with all the other interfaces.

We should either take `AccountAddress` for everything, or `HexInput` for everything.

Reply:
Sounds good, will update to change this back to `HexInput` for now. 

Though personally, I think AccountAddress make more sense here. We can circle back later on

Reply:
Fixed